x-pack/solutions/security/plugins/security_solution/common/experimental_features.ts

```diff
@@ -266,9 +266,11 @@ type Mutable<T> = { -readonly [P in keyof T]: T[P] };
 
 const allowedKeys = Object.keys(allowedExperimentalValues) as Readonly<ExperimentalConfigKeys>;
 
+const disableExperimentalPrefix = 'disable:' as const;
+
 /**
  * Parses the string value used in `xpack.securitySolution.enableExperimental` kibana configuration,
- * which should be a string of values delimited by a comma (`,`)
+ * which should be an array of strings.
```

Comment:
(Optional)
Maybe add info. here about the the `disabled:` token and how it can be used via the `yaml` configuration file to disable features?